### PR TITLE
Test items counter Implementation

### LINKED ISSUE #3: [3pt]  - Add tests for items counter - Student A
Make sure that function for the items counter is covered with unit tests using Jest. Make sure that you have tests for all edge cases.

### PR DESCRIPTION
**Pull request by cancelei**
Hello Amir,

In this part I deliver the Homescreen Counter + The Jest Test Suite.

You should see the counter of pokemon on the screen in the first item of the Nav bar.

Test should pass, as evidenced below:

image

Important to note => File changes seem to be big due to dist update.
I created a new Variable MAXCARDS to receive the number of cards that we want to appear on the Main page. That also helps me when implementing the test strategy in this PR.

In my next PR, you will receive an improved version of the counter JS algo.

### CODE REVIEW

File: src/homepagecounter.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export function likecount() {
+  // const likebutton = document.querySelectorAll('.likebutton');
+  // const likebuttoncounter = document.querySelectorAll('.likebuttoncounter');
+
+  // likebutton.forEach((button) => {
+  //   button.addEventListener('click', () => {
+  //     const likebuttoncounter = button.nextElementSibling;
+  //     likebuttoncounter.innerHTML = 'Liked';
+  //   });
+  // });
```
Comment (Salarhabibi-amir) [inline src/homepagecounter.js:17]: It would be nice if you do not push the commented code to the repository. Please remove this part.

**Review by Salarhabibi-amir — APPROVED** [review]
## Hi @cancelei , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 




## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Linters are green ✅ 
- Correct `GitHub` workflow ✅ 
- Meaningful **Github** commits ✅  
- Correct `.github/workflows` for linters ✅ 
- Followed *best-practices* ✅ 
- Correct **Semantics** used ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
